otp.sh

```diff
@@ -9,7 +9,7 @@
 #openssl enc -aes-256-cbc -d -salt -in file.txt.enc -out file.txt
 
 # Init
-TOKENFILES_DIR="$( dirname ${0} )/tokenfiles"
+TOKENFILES_DIR=${BASH_OTP_TOKENFILES_DIR:-"$( dirname ${0} )/tokenfiles"}
```

Comment:
needs to have the double quotes outside the ${} in case the BASH_OTP_TOKENFILES_DIR has spaces in it